### PR TITLE
feat: additional features for attachments

### PR DESCRIPTION
**Pull request by mikitabut**
https://github.com/epam/ai-dial-chat/issues/211

Next cases covered:
Separate Modal to manage files which opened from left side bar
Map mime types to extensions on UI
Bulk remove
Bulk download

### CODE REVIEW

**Comment by mikitabut** [comment]
/deploy-review
>Deployment status: success

File: src/components/Files/PreUploadModal.tsx [hunk 1]
```diff
@@ -98,25 +110,20 @@ export const PreUploadDialog = ({
...
+        512 * bytesInMb,
+      ).map((file) => file.name);
+      const incorrectTypeFiles: string[] = getFilesWithInvalidFileType(
+        files,
+        allowedTypes,
+      ).map((file) => file.name);
+      const filteredFiles = files.filter(
+        (file) =>
+          !incorrectSizeFiles.includes(file.name) &&
+          !incorrectTypeFiles.includes(file.name),
```
Comment (IlyaBondar) [inline src/components/Files/PreUploadModal.tsx:124]: wrap `incorrectSizeFiles` and `incorrectTypeFiles` into `new Set` to search with O(1)
```suggestion
          !incorrectSizeFiles.has(file.name) &&
          !incorrectTypeFiles.has(file.name),
```

File: src/pages/api/files/index.ts [hunk 2]
```diff
@@ -85,8 +85,8 @@ async function handleGetRequest(
 ) {
   const { path } = req.query as { path: string };
   const url = `${process.env.OPENAI_API_HOST}/v1/files${
-    path && `/${encodeURI(path)}`
-  }?path=absolute&purpose=content`;
+    path && encodeURI(path)
```
Comment (IlyaBondar) [inline src/pages/api/files/index.ts:88]: can path be empty? And if yes we will inject "false" into url in this case

**Review by IlyaBondar — APPROVED** [review]

File: src/pages/api/files/index.ts (hunk 2 above)
Comment (mikitabut) [reply src/pages/api/files/index.ts:88]: By design it can't be empty, because every file has path and it's passing to URL
But I will add some checks

**Pull request merged**